javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var newArr = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      delete null;
```

Comment:
this line is not actually doing anything since `null` has not connection to the array, but it does work for the proper output because the else statement is pushing for all indexes that are not null